packages/wdio-utils/src/duration.ts

```diff
@@ -0,0 +1,44 @@
+class DurationTracker {
+    private timers: Record<string, number> = {}
+    private durations: Record<string, number> = {}
```

Comment:
Since the `getSummary` function explicitly expects certain types of duration I think it makes sense to apply some stricter typing here.